### ISSUE TITLE
[BUG] Additional startup arguments includes pwd

### ISSUE THREAD

**Issue by daviehh** [JunoLab/Juno.jl#244]
Issue: current working directory get passed as additional startup argument

### Steps to reproduce

1. Set additional startup argument to something like 
```
-J "/Users/[username]/.julia/packages/PackageCompiler/oT98U/sysimg/sys.dylib"
``` 
to use the custom sysimg compiled with packagecompiler

2. 
```
ERROR: could not load library "/path/to/current/directory/ '/Users/[username]/.julia/packages/PackageCompiler/oT98U/sysimg/sys.dylib'"
dlopen(/path/to/current/directory/ '/Users/[username]/.julia/packages/PackageCompiler/oT98U/sysimg/sys.dylib'.dylib, 6): image not found
```

**Comment by pfitzseb** [comment]
I've improved the description for that setting in [JunoLab/atom-julia-client@f9d16cf](https://github.com/JunoLab/atom-julia-client/commit/f9d16cf1aa89eabe16504bb706ff2439dd134a54).

TL;DR: Don't quote paths (or anything, really) in that setting. There also shouldn't be any whitespace between `-J` and the path.
